src_py/cursors.py

```diff
@@ -28,7 +28,8 @@
     arrow, diamond, ball, broken_x, tri_left, tri_right
 
 There is also a sample string cursor named 'thickarrow_strings'.
-The compile() function can convert these string cursors into cursor byte data.
+The compile() function can convert these string cursors into cursor byte data that can be used to
+create Cursor objects (which can also be instantiated from surfaces).
```

Comment:
I think this change makes things more confusing.

Reply:
Ok, I will try to make it clearer